### ISSUE TITLE
why

### ISSUE THREAD

**Issue by Nchu-hei** [hufo1/STFNBRPS#3]
class ConvTemporalGraphical(nn.Module):

    def __init__(self,
                 in_channels,
                 out_channels,
                 kernel_size,
                 t_kernel_size=1,
                 t_stride=1,
                 t_padding=0,
                 t_dilation=1,
                 bias=True):
        super().__init__()

        self.kernel_size = kernel_size
        self.conv = nn.Conv2d(
            in_channels,
            out_channels * kernel_size,
            kernel_size=(t_kernel_size, 1),
            padding=(t_padding, 0),
            stride=(t_stride, 1),
            dilation=(t_dilation, 1),
            bias=bias)

    def forward(self, x, L):
       
        x = torch.matmul(L, x.permute(0, 1, 3, 2))

        # Apply ReLU activation
        x = F.relu(x.permute(0, 1, 3, 2))

        return x   这个模型为什么没有调用卷积
